### ISSUE TITLE
initial build with select example fails

### ISSUE THREAD

**Issue by Dierk** [paf31/purescript-sdom#8]
building and running the counter example works fine but the select example gives the error below
```
> * Building project in /projects/git/purescript-sdom/examples/select
> Compiling Main
> Error found:
> in module Main
> at src/Main.purs line 53, column 9 - line 53, column 26
> 
>   Unknown value Components.select
```

**Comment by paf31** [comment]
Can you please try now? I published 0.1.2.

**Comment by Dierk** [comment]
works fine, thanks!
